Src/ILGPU/Runtime/OpenCL/API/CLAPI.cs

```diff
@@ -463,7 +463,7 @@ public static CLError CreateContext(IntPtr device, out IntPtr context)
             IntPtr device,
             out string buildLog)
         {
-            const int LogSize = 4096;
+            const int LogSize = 32_000;
```

Comment:
Since you have increased the log size to about 32Kb, it might be a good idea to allocate it on the heap instead. Can you adjust the following line to use a heap-allocated array?
After that, this PR should be ready.